core/src/main/kotlin/behavior/interaction/ApplicationCommandInteractionBehavior.kt

```diff
@@ -38,8 +39,8 @@ internal fun ApplicationCommandInteractionBehavior(
 
 }
 
-public inline suspend fun ApplicationCommandInteractionBehavior.modal(title: String, customId: String,builder: ModalBuilder.() -> Unit): PublicInteractionResponseBehavior {
+public inline suspend fun ApplicationCommandInteractionBehavior.modal(title: String, customId: String,builder: ModalBuilder.() -> Unit): PopupInteractionResponseBehavior {
     contract { callsInPlace(builder, InvocationKind.EXACTLY_ONCE) }
      kord.rest.interaction.createModalInteractionResponse(id, token,title, customId, builder)
```

Comment:
```suggestion
public suspend inline fun ApplicationCommandInteractionBehavior.modal(title: String, customId: String, builder: ModalBuilder.() -> Unit): PopupInteractionResponseBehavior {
    contract { callsInPlace(builder, InvocationKind.EXACTLY_ONCE) }
     kord.rest.interaction.createModalInteractionResponse(id, token, title, customId, builder)
```
Non-canonical modifiers order